deltachat-ios/Controller/NewChatViewController.swift

```diff
@@ -15,8 +15,9 @@ class NewChatViewController: UITableViewController {
     private let newOptions: [NewOption]
 
     private let sectionNew = 0
-    private let sectionContacts = 1
-    private let sectionsCount = 2
+    private let sectionInviteFriends = 1
```

Comment:
i am wondering if it isn't better to have "Invite Friends" at the end of the list:

- for new user without contacts or with few contacts it is as easy accessible as before

- however, when the list gets filled, it gets out of the way - the same time, the list is filled and "Invite Friends" obviously works. and, it is still clear where the option is: at the end of the list

Reply:
Thanks for your feedback, I thought about that, too, and will change it to be that way.

Originally I thought it might be a good idea to nudge people to invite their friends, but you're totally right: What happens if all their friends already use Deltachat? So I'll put it at the end :)